README.md

```diff
@@ -3,12 +3,24 @@
 [![Quality Gate](https://sonarcloud.io/api/project_badges/measure?project=io.apicollab%3Afs%3Auploader&metric=alert_status)](https://sonarcloud.io/dashboard?id=io.apicollab%3Afs%3Auploader)
 
 This module makes it easy to upload new or updated api swagger spec to api portal.
-
 # Prerequisites
 - Java 8
 - Maven
 - Git
-
+# Mandatory custom property in Swagger/OpenAPI spec
+Every API spec must be linked to an existing application. For the uploader module to upload API spec under specific application, a custom property `x-app-id` must be made available under `info` section of spec to be uploaded. Refer to sample snippet below:
+```
+openapi: "3.0.0"
+info:
+  version: 1.0.0
+  title: API Collab - Server
+  x-app-id: 1 # Custom property for application id
```

Comment:
Its more user friendly to call it x-app-name and have the name of the application?

Reply:
Agree but this might need few changes on portal's API.

Created #2 for this change.